Pin to PL 0.29 (#134)
PennyLaneAI/pennylane-sf: README.rst

`Strawberry Fields <https://strawberryfields.readthedocs.io>`__ is a full-stack Python library
for designing, simulating, and optimizing photonic quantum circuits.

.. warning::

>     This plugin will not be supported in newer versions of Pennylane. It is compatible with versions

Author: ```suggestion
    This plugin will not be supported in newer versions of PennyLane. It is compatible with versions
```

>     This plugin will not be supported in newer versions of PennyLane. It is compatible with versions

Author: ```suggestion
    This plugin is no longer maintained and is not compatible with the latest version of PennyLane.
```

>     of PennyLane up to and including 0.29.

Author: ```suggestion
    It is supported for version 0.29 of PennyLane and below.
```

> 

Reviewer: Thanks @lillian542! Please could we also have this callout in the docs page, i.e., [here](https://docs.pennylane.ai/projects/strawberryfields/en/latest/) just below the release.

Author: The first 3 lines of that page are pulled from this file, putting it here should put it both places:

```
:Release: |release|

.. include:: ../README.rst
  :start-after:	header-start-inclusion-marker-do-not-remove
  :end-before: header-end-inclusion-marker-do-not-remove
```

Author: Not at the very top though, I can rearrange the order

Author: https://xanaduai-pennylane--134.com.readthedocs.build/projects/strawberryfields/en/134/
